src/haxelib/client/Main.hx

```diff
@@ -167,7 +166,7 @@ class Main {
 		apiVersion : "3.0",
 		noSsl : false
 	};
-	static var IS_WINDOWS = (Sys.systemName() == "Windows");
+	public static var IS_WINDOWS = (Sys.systemName() == "Windows");
```

Comment:
Should be final

Reply:
`final` is not available in haxe 3, and `inline var` gives an error saying `(Sys.systemName() == "Windows")` is not a constant variable.

Another option is to change it back to private and just copy this line so that RepoManager has its own copy, like FsUtils does, but maybe it's better to come up with a better solution altogether.

Reply:
It can be `var IS_WINDOWS(default,nul) = ...`

Reply:
If we're breaking compatibility with the Exception would it be fine to just use `final`?

Reply:
Yes